From: Guilherme Puida Moreira <puida@debian.org>
Date: Thu, 7 May 2026 00:00:00 +0000
Subject: Disable StrictModes in test sshd

We don't care about permissions when running sshd inside autopkgtests.
This patch makes it possible to run ssh-related pytest tests.

Forwarded: no
---
 tests/http/testenv/sshd.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/http/testenv/sshd.py b/tests/http/testenv/sshd.py
index e800ced..00ad206 100644
--- a/tests/http/testenv/sshd.py
+++ b/tests/http/testenv/sshd.py
@@ -271,6 +271,7 @@ class Sshd:
             'PasswordAuthentication no',
             # in CI, we might run as root, allow this
             'PermitRootLogin yes',
+            'StrictModes no',
             'LogLevel VERBOSE',
             f'AuthorizedKeysFile {self._auth_keys}',
             f'PidFile {self._pid_file}',
